src/firefly/js/visualize/ui/MouseReadoutOptionPopups.jsx

```diff
@@ -22,18 +22,24 @@ import { dispatchChangeReadoutPrefs} from '../../visualize/MouseReadoutCntlr.js'
 import {dispatchShowDialog, dispatchHideDialog} from '../../core/ComponentCntlr.js';
 import {primePlot} from '../PlotViewUtil.js';
 import {visRoot} from '../ImagePlotCntlr.js';
-import {isHiPS} from '../WebPlot.js';
+import {isCelestialImage, isHiPS} from '../WebPlot.js';
 
 //define the labels and values for the radio options
-const coordOptions= [
+const celestialCoordOptions= [
 	{label: 'Equatorial J2000 HMS', value: 'eqj2000hms'},
 	{label: 'Equatorial J2000 decimal', value: 'eqj2000DCM' },
 	{label: 'Galactic', value: 'galactic'},
 	{label: 'Equatorial B1950', value: 'eqb1950'},
 	{label: 'Ecliptic J2000', value: 'eclJ2000'},
 	{label: 'Ecliptic B1950', value: 'eclB1950'},
 	{label: 'FITS Image Pixel', value: 'fitsIP'},
-    {label: 'Zero based Image Pixel', value: 'zeroIP'}
+	{label: 'Zero based Image Pixel', value: 'zeroIP'}
+];
+
+const noncelestialCoordOptions= [
+	{label: 'WCS Coords', value: 'wcsCoords'},
```

Comment:
I think we want this to by dynamic. so in the case with will say XANG,YANG. That means you will have to pass the wcs label to the dialog, I think

Reply:
Done.